packages/flutter/lib/src/rendering/sliver.dart

```diff
@@ -89,6 +89,21 @@ Axis axisDirectionToAxis(AxisDirection axisDirection) {
   return null;
 }
 
+/// Returns the [AxisDirection] in which reading occurs in the given [TextDirection].
+///
+/// Specifically, returns [AxisDirection.left] for [TextDirection.rtl] and
+/// [AxisDirection.right] for [TextDirection.ltr].
+AxisDirection textDirectionToAxisDirection(TextDirection textDirection) {
+  assert(textDirection != null);
+  switch (textDirection) {
+    case TextDirection.rtl:
+      return AxisDirection.left;
+    case TextDirection.ltr:
+      return AxisDirection.right;
+  }
+  return null;
```

Comment:
@lukechurch This is the kind of thing for which I really wish I could put methods on enums.